src/ConfigPostProcessor.php

```diff
@@ -22,7 +22,7 @@
 class ConfigPostProcessor
 {
     /** @internal */
-    const SERVICE_MANAGER_KEYS_OF_INTEREST = [
+    private const SERVICE_MANAGER_KEYS_OF_INTEREST = [
```

Comment:
Sadly, you are introducing BC break when you add return typehints to public methods on non-final classes or change visibility of previously public constants.
I'd suggest to remove typehints for this PR in order to avoid regressions.

Reply:
Sorry about that… I thought it would be OK being marked as internal 🤦‍♂️

Reply:
Its not just this constant, there are plenty of other changes which introduce bc breaks.
Even tho its internal we cannot trust userland code here. ;-)

Reply:
Gotcha. For now, I've just killed of all of the changes in test and src so these can be introduced in a separate PR if required